doc/rbd/iscsi-target-ansible.rst

```diff
@@ -0,0 +1,360 @@
+|                                      | it will be removed from the iSCSI    |
+|                                      | gateway and deleted from the         |
+|                                      | configuration.                       |
++--------------------------------------+--------------------------------------+
+| ``client_connections``               | This section defines the iSCSI       |
+|                                      | client connection details together   |
+|                                      | with the LUN (RBD image) masking.    |
+|                                      | Currently only CHAP is supported as  |
+|                                      | an authentication mechanism. Each    |
+|                                      | connection defines an ``image_list`` |
+|                                      | which is a comma separated list of   |
+|                                      | the form                             |
+|                                      | ``pool.rbd_image[,pool.rbd_image,…​] |
+|                                      | ``.                                  |
+|                                      | RBD images can be added and removed  |
+|                                      | from this list, to change the client |
+|                                      | masking. Note that there are no      |
+|                                      | checks done to limit RBD sharing     |
+|                                      | across client connections.           |
++--------------------------------------+--------------------------------------+
+
+.. NOTE::
+  When using the ``gateway_iqn`` variable, and for Red Hat Enterprise Linux
+  clients, installing the ``iscsi-initiator-utils`` package is required for
+  retrieving the gateway’s IQN name. The iSCSI initiator name is located in the
+  ``/etc/iscsi/initiatorname.iscsi`` file.
+
+.. NOTE::
+  If using previously configured RBD images, then verify that the following
+  features are enabled:
+
+  -  ``layering``
+
+  -  ``exclusive-lock``
+
+  If these features are not enable, then RBD mapping failures will occur across
+  the iSCSI gateway nodes. Also, if enabled, disable these features:
+
+  -  ``object-map``
+
+  -  ``fast-diff``
+
+  -  ``deep-flatten``
+
+**Deploying:**
+
+On the Ansible installer node, perform the following steps.
+
+1. As ``root``, execute the Ansible playbook:
+
+   ::
+
+       # cd /usr/share/ceph-ansible
+       # ansible-playbook ceph-iscsi-gw.yml
+
+   .. NOTE::
+    The Ansible playbook will handle RPM dependencies, RBD creation
+    and Linux IO configuration.
+
+   .. WARNING::
+    On stand-alone iSCSI gateway nodes, verify that the correct Red
+    Hat Ceph Storage 2 software repositories are enabled. If they are
+    disabled or unavailable, the wrong packages will be installed.
+
+2. Verify the configuration:
+
+   ::
+
+       # targetcli ls
```

Comment:
Nit: probably should note that this should be done on a gateway host, not the ansible host. Also might just say use "gwcli ls" and link to the gwcli doc (?)